src/components/LanguageSwitcher.svelte

```diff
@@ -40,6 +40,9 @@
         <a on:click={menuToggle} href="{`fr/${pathWithoutLang}`}"
            class="uppercase block px-2 py-1 text-gray-700 font-semibold rounded hover:bg-teal-500 hover:text-white"
            class:selected="{lang === 'fr' ? 'selected' : ''}"> {flag('fr')} fr</a>
+        <a on:click={menuToggle} href="{`it/${pathWithoutLang}`}"
+           class="uppercase block px-2 py-1 text-gray-700 font-semibold rounded hover:bg-teal-500 hover:text-white"
+           class:selected="{lang === 'it' ? 'selected' : ''}"> {flag('it')} fr</a>
```

Comment:
there´s a small mistake. the `fr` at the end needs to be changed to `it`

Reply:
ops.... copy&paste mistake